config/config.exs

```diff
@@ -139,7 +139,7 @@ config :skate, Oban,
 
 # Configures Elixir's Logger
 config :logger, :console,
-  format: "$time $metadata[$level] $message\n",
+  format: "$time [$level] $metadata$message\n",
```

Comment:
Curiosity: How come we don't need a space between `$metadata` and `$message`?

Reply:
The `$metadata` variable has a space appended to the end of it apparently. Which is also why a space wasn't needed between `$metadata[$level]`

> `$metadata` - user controlled data presented in `"key=val key2=val2 "` format
> – https://hexdocs.pm/logger/1.17.2/Logger.Formatter.html#module-formatting